### ISSUE TITLE
Handle events - React in Vue.

### ISSUE THREAD

**Issue by cortez-scr** [akxcv/vuera#85]
Hello, Sorry if this question is duplicated but I couldn't find an answer yet.
I hope you can help me.
Let say in my Vue App I have a react component with a label.
This label looks something like this:

```
render() {
    		return (
			<b>This title is a prop from react: {this.props.title}</b>
			)
		}
```
I import this react component into Vue like this:
```
<template>
  <span>
      <my-react-component :title="title"></my-react-component >
  </span>
</template>


<script>
import { MyReactComponent } from './MyReactComponent.jsx'

export default {
  name: "App",
  components: {
    MyReactComponent 
  },
  data () {
    return { title: "Title set by Vue" };
  }
};

</script>
```
This will set the {this.props.title} in react perfectly.
So far so good, now I would like to know how to make a Button in Vue reset the title.
In other words by clicking a button in Vue I want to change the {this.props.title} in react.

I tried the following but I could not make it work.

```
<template>
  <span>
      <my-react-component :title="title"></my-react-component >
	  <b-button @click="reset">+</b-button> 
  </span>
</template>

<script>
import { MyReactComponent } from './MyReactComponent .jsx'
export default {
  name: "App",
  components: {
    MyReactComponent 
  },
  data () {
    return { title: "Title set by Vue" };
  },
   methods: {
    reset () {
      this.title = "Title reset by Vue"
    }
  }
};
</script>

```

Thanks in advance and best regards!
